src/main/java/application/controllers/TreeController.java

```diff
@@ -59,32 +44,8 @@ public TreeController(MainController m, String string) {
         init();
     }
 
-    /**
-     * Set-up the tree model from a Newick data file.
-     *
-     * @return A Newick tree.
-     */
-    @SuppressFBWarnings({"I18N", "NP_DEREFERENCE_OF_READLINE_VALUE"})
-    public Tree getTreeFromFile(String s) {
-        InputStream stream = this.getClass().getResourceAsStream("/340tree.rooted.TKK.nwk");
```

Comment:
This was a duplicate unused method, although this is still hardcoded in the PhylogeneticTree class.